cypress/integration/InstancesTest.ts

```diff
@@ -64,6 +64,82 @@ describe("Instances Test", () => {
     });
   });
 
+  describe("the Instance statuses", () => {
+    beforeEach(() => {
+      cy.visit("/");
+      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
+        "not.exist"
+      );
+    });
+
+    it("Accepted", () => {
+      const instanceName: string = "Instance three";
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId(instanceName, "PF4/TableRow")
+        .should("be.visible")
+        .within(() => {
+          cy.get("td:first").should("have.html", instanceName);
+          cy.get("td:nth-child(2)").should("have.text", "accepted");
+          cy.get("td:nth-child(4)")
+            .click()
+            .within(() => {
+              cy.ouiaType("PF4/DropdownItem").then(($items) => {
+                expect($items.eq(0)).have.text("Details");
+                expect($items.eq(0)).be.enabled;
+                expect($items.eq(1)).have.text("Delete");
+                expect($items.eq(1)).have.attr("aria-disabled", "true");
```

Comment:
Just asking, can't we use `expect($items.eq(0)).be.disabled;`? as I see in other test `be.enabled`

Reply:
@jomarko I have tried it but it failed. I did not found what `.be.disabled` tested. There is the aria-disabled attribute instead of the disabled attribute.